dmcontent/content_loader.py

```diff
@@ -249,6 +261,41 @@ def get_previous_question_id(self, question_id):
 
         return None
 
+    def get_next_question_slug(self, question_slug=None):
```

Comment:
It is, and it would be good to do that work, but I think it's relied upon elsewhere in the code and I don't feel we have time to dig it out right now. I intended to add it to tech debt board but don't have write access.